cli/flox/doc/flox-activate.md

```diff
@@ -126,6 +133,9 @@ See [`manifest.toml(5)`](./manifest.toml.md) for more details on shell hooks.
     This is currently an implementation detail
     and its contents are subject to change.
 
+`$FLOX_ACTIVATE_START_SERVICES`
+:   `"true"` if this activation started services, `"false"` otherwise.
```

Comment:
(non-blocking observation) This cements the env var as user-facing in relation [this Slack thread](https://flox-dev.slack.com/archives/C06T9RL3BQT/p1723503905612429). Which I don't have strong feelings for either way but we'll need to bear it in mind if we change/remove it in future.